src/Geta.NotFoundHandler.Admin/wwwroot/GetaNotFoundHandlerAdmin/js/dashboard.js

```diff
@@ -36,6 +35,25 @@
         });
     }
 
+    function adjustModalPosition() {
+        var modalTriggers = document.querySelectorAll('.modal-trigger');
+        modalTriggers.forEach(function (modalTrigger) {
+            modalTrigger.addEventListener('click',
+                function () {
```

Comment:
Please reduce nesting in this function. 
![image](https://user-images.githubusercontent.com/646298/219621798-e135b3c8-eaa7-474b-8994-c50be634c951.png)

Checks for `modalTrigger.dataset.bsTarget` and `modalDialog` can return as it finds that an element does not exist.

Reply:
Should I update to closest release candidate to this beta of bootstrap as well still?

Reply:
I've updated it already.